cmd/bosun/sched/sched.go

```diff
@@ -540,6 +540,12 @@ const (
 
 func init() {
 	metadata.AddMetricMeta("bosun.statefile.size", metadata.Gauge, metadata.Bytes, "The total size of the bosun state file.")
+	metadata.AddMetricMeta("bosun.check.duration", metadata.Gauge, metadata.Second, "The number of seconds it took Bosun to check each alert rule.")
```

Comment:
Existing line wasn't word wrapped. When we have multiple conventions I do my best to follow all of them :-P